java/Dockerfile

```diff
@@ -15,16 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM maven
+FROM maven:3.6-jdk-8
```

Comment:
So the main objective for this dockerfile is to quickly (re-)execute java tests.

Reply:
Fixing the maven version is probably a good idea